### ISSUE TITLE
Use Swift's (De)Codable Interfaces instead of SwiftyJSON

### ISSUE THREAD

**Issue by RobinThrift** [xzzz9097/Swiftify#1]
Since Swift 4 can now natively de- and encode JSON it might be a nice way to reduce dependencies, using the built in methods going forward. I'd be happy to help if you need assistance :)

**Comment by xzzz9097** [comment]
Thanks for the input! I'm doing it right now in swift4 branch.
Decodable structs are ready in SpotifyItems.swift, I've adapted 'find' function for it, now I have to do the same for the 'library' func and I'll be able to remove the SwiftyJSON initialization for the structs.
Right now I'm also using SwiftyJSON to read/write application and token data to JSON files, but I've also added storing in a preference as an option so I may drop that altogether.
